core/src/main/scala/org/bitcoins/core/wallet/builder/RawTxFinalizer.scala

```diff
@@ -0,0 +1,169 @@
+package org.bitcoins.core.wallet.builder
+
+import org.bitcoins.core.currency.{CurrencyUnit, Satoshis}
+import org.bitcoins.core.number.{Int32, UInt32}
+import org.bitcoins.core.policy.Policy
+import org.bitcoins.core.protocol.script.ScriptPubKey
+import org.bitcoins.core.protocol.transaction._
+import org.bitcoins.core.wallet.fee.FeeUnit
+import org.bitcoins.core.wallet.utxo.{InputInfo, InputSigningInfo}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Success, Try}
+
+sealed trait RawTxFinalizer {
```

Comment:
It would be nice if we could nest finalizers, so we can use many at once without needing a type for every combination.